srcpkgs/kgpg/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'kgpg'
+pkgname=kgpg
+version=19.08.1
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules"
+makedepends="kwindowsystem-devel kcrash-devel kdbusaddons-devel
+ kiconthemes-devel kjobwidgets-devel kio-devel
+ kservice-devel ktextwidgets-devel kxmlgui-devel
+ kwidgetsaddons-devel knotifications-devel kdoctools-devel
+ akonadi-contacts-devel kcontacts-devel boost-devel"
+depends="akonadi-contacts phonon-qt5-backend-gstreamer oxygen gpgme"
+short_desc="Simple interface for GnuPG, a powerful encryption utility"
+maintainer="k4leg <d0xi@inbox.ru>"
+license="GPL-2.0-only"
+homepage="https://kde.org/applications/utilities/kgpg/"
+distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=441a0bfa58df14bad87f5f446b89113dc20365424f6a87aec30125c9221815c5
+
+if [ "$CROSS_BUILD" ]; then
+ hostmakedepends+=" qt5-devel kcoreaddons-devel kdoctools kconfig"
```

Comment:
Does it really need `qt5-devel` to cross-compile? Usually having `qt5-host-tools` and `qt5-qmake` is enough.

Reply:
Yes, it compiles with qt5-qmake, thanks.